frontend/src/views/web/challenge/edit-challenge/edit-challenge-evaluation-criteria.html

```diff
@@ -21,7 +21,8 @@
                                      trix-attachment-remove="trixAttachmentRemove(e, editor);"
                                      prevent-trix-file-accept = "true"
                                      placeholder="Evaluation Criteria">
-                          </trix-editor>
+                          <trix-editor>
```

Comment:
Why do you remove the closing </trix-editor> and start a new one?

Reply:
There is no closing tag for either of them.

Reply:
Typo